### PR TITLE
Feature/mintr refetches

### PR DESCRIPTION
**Pull request by andytcf**
Implement refetching on all mutative contract calls (15 second buffer to account for event updates)

### CODE REVIEW

File: src/helpers/notifyHelper.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import { getEtherscanTxLink } from 'helpers/explorers';
+
+export function notifyHandler(notify, hash, networkId, callback, message) {
+	let { emitter } = notify.hash(hash);
+	emitter.on('txConfirmed', result => {
+		setTimeout(() => {
+			callback();
+		}, 15000);
```
Comment (clementbalestrat) [inline src/helpers/notifyHelper.js:8]: Have you tried with a lower timeout? like 5 or 10 secs?

`txConfirmed` is when the transaction is mined so we shouldn't need to wait 15 secs to get the updated data.

**Review by clementbalestrat — CHANGES_REQUESTED** [review]
@andytcf this is great man. Well done. A few minor stuff to update and then I'll merge it :)

**Comment by clementbalestrat** [comment]
@andytcf also if you could fix the merge conflict

File: src/helpers/notifyHelper.js (hunk 1 above)
Comment (andytcf) [reply src/helpers/notifyHelper.js:8]: tried the lower timeouts but 15 sec seems to be optimum for the event to be picked up, not entirely sure why so would welcome alternatives here

Comment (clementbalestrat) [reply src/helpers/notifyHelper.js:8]: Yeah I guess it's because of Infura

**Pull request merged**